### PR TITLE
Create manhattan.lua

### PR DESCRIPTION
**Pull request by Claude477**
Manhattan-map with crashsite.

### CODE REVIEW

File: map_gen/maps/crash_site/presets/manhattan.lua [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+local world_map = b.picture(pic)
+local degrees = require "utils.math".degrees
+
+local x_offset, y_offset = 200, 1337
+world_map = b.translate(world_map, x_offset, y_offset)
+world_map = b.rotate(world_map, degrees(-270))
+
+local scale = 2
+local height = 8000 * scale
+local width = 8000 * scale
```
Comment (grilledham) [inline map_gen/maps/crash_site/presets/manhattan.lua:18]: Why 8000 for the height and width?
In the world preset  these numbers come from the develop/map_gen/data/presets/world-map height and width values.
Same question for the other two pull requests.

File: map_gen/maps/crash_site/presets/manhattan.lua [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+local config = {
+    scenario_name = 'crashsite-manhattan',
+    map_gen_settings = {
+        MGSP.starting_area_very_low,
+        MGSP.ore_oil_none,
+        MGSP.enemy_none,
+        MGSP.cliff_none
+    },
+    grid_number_of_blocks = 15,
+    mini_grid_number_of_blocks = 29,
```
Comment (grilledham) [inline map_gen/maps/crash_site/presets/manhattan.lua:35]: We might be able to use smaller numbers for grid_number_of_blocks and mini_grid_number_of_blocks depending on map size.
If I remember correctly a good value for grid_number_of_blocks is floor(map_size/180)
and for mini_grid_number_of_blocks floor(map_size/96)
Where map size is the larger of width or height after scaling.

Same for the other two pull requests

**Review by grilledham — COMMENTED** [review]
Some suggestions for change, but they would need to be tested in cases I'm mistaken.

File: map_gen/maps/crash_site/presets/manhattan.lua (hunk 1 above)
Comment (Claude477) [reply map_gen/maps/crash_site/presets/manhattan.lua:18]: normal map meybe, too small so i make it bigger

File: map_gen/maps/crash_site/presets/manhattan.lua (hunk 2 above)
Comment (Claude477) [reply map_gen/maps/crash_site/presets/manhattan.lua:35]: ok that one i dont know about it i just copyed in crashsite default-configs

File: map_gen/maps/crash_site/presets/manhattan.lua (hunk 1 above)
Comment (Claude477) [reply map_gen/maps/crash_site/presets/manhattan.lua:18]: height = 2979,
width = 898 was original settings so yeah is can make put 2x larger size..

Comment (Claude477) [reply map_gen/maps/crash_site/presets/manhattan.lua:18]: height=5958
width = 1796 with scale 2

Comment (grilledham) [reply map_gen/maps/crash_site/presets/manhattan.lua:18]: Scaling the map is fine, but these values and used to make the bounds_shape which determines where outposts can be made, it shouldn't need to be bigger than the map itself.

Comment (Claude477) [reply map_gen/maps/crash_site/presets/manhattan.lua:18]: local x_offset, y_offset = 200, 1337 if change it need reset default reswap location ohter place..

Comment (Claude477) [reply map_gen/maps/crash_site/presets/manhattan.lua:18]: ahh i see now 5958x1796(map) and bound size(16000x16000) yeah bound size can make as map size sound good..

Comment (Claude477) [reply map_gen/maps/crash_site/presets/manhattan.lua:18]: local scale = 2
local height = 3000 * scale
local width = 1000 * scale sound good solution ?

Comment (Claude477) [reply map_gen/maps/crash_site/presets/manhattan.lua:18]: as bound size ?

Comment (Claude477) [reply map_gen/maps/crash_site/presets/manhattan.lua:18]: image
 hmm need still fix it T_T

Comment (Claude477) [reply map_gen/maps/crash_site/presets/manhattan.lua:18]: need make x location 0->8000 bound size..

Comment (Claude477) [reply map_gen/maps/crash_site/presets/manhattan.lua:18]: ok trying fix bound size..

File: map_gen/maps/crash_site/presets/manhattan.lua (hunk 2 above)
Comment (Claude477) [reply map_gen/maps/crash_site/presets/manhattan.lua:35]: ok found out in crashsite no need that blocks

File: map_gen/maps/crash_site/presets/manhattan.lua (hunk 1 above)
Comment (Claude477) [reply map_gen/maps/crash_site/presets/manhattan.lua:18]: image
found out that 8000x8000 is best in that map for fill all map with bitters tryed 3k x 5958 geted by image

**Pull request closed without merging**